### PR TITLE
[extractor/youtube] Add the first 3 characters of client name in `format_note`

### PR DESCRIPTION
**Pull request by Lesmiscore**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

This PR adds the client names to `format_note` returned by the Youtube extractor for convenience
If you wish to enforce this to all the formats (MPD and HLS), a little rewrite would be required

p.s. make a release before reviewing this

```
$ ./yt-dlp.sh -s test:youtube_2 --extractor-args youtube:player_client=all -F
[TestURL] Extracting URL: test:youtube_2
[TestURL] Test URL: https://www.youtube.com/watch?v=BaW_jenozKc&v=yZIXLfi8CZQ
[youtube] Extracting URL: https://www.youtube.com/watch?v=BaW_jenozKc&v=yZIXLfi8CZQ
(snip)
[info] Available formats for BaW_jenozKc:
ID      EXT  RESOLUTION FPS CH │   FILESIZE   TBR PROTO │ VCODEC          VBR ACODEC      ABR ASR MORE INFO
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
233     mp4  audio only        │                  m3u8  │ audio only          unknown             Default
234     mp4  audio only        │                  m3u8  │ audio only          unknown             Default
249-drc webm audio only      2 │   62.24KiB   52k https │ audio only          opus        52k 48k low, DRC, TVH, webm_dash
250-drc webm audio only      2 │   80.49KiB   67k https │ audio only          opus        67k 48k low, DRC, TVH, webm_dash
139     m4a  audio only      2 │   58.59KiB   48k https │ audio only          mp4a.40.5   48k 22k low, AND, m4a_dash
249     webm audio only      2 │   58.17KiB   48k https │ audio only          opus        48k 48k low, AND, webm_dash
250     webm audio only      2 │   76.07KiB   63k https │ audio only          opus        63k 48k low, AND, webm_dash
140-drc m4a  audio only      2 │  157.22KiB  130k https │ audio only          mp4a.40.2  130k 44k medium, DRC, TVH, m4a_dash
251-drc webm audio only      2 │  138.80KiB  115k https │ audio only          opus       115k 48k medium, DRC, TVH, webm_dash
140     m4a  audio only      2 │  154.06KiB  128k https │ audio only          mp4a.40.2  128k 44k medium, AND, m4a_dash
251     webm audio only      2 │  138.96KiB  116k https │ audio only          opus       116k 48k medium, AND, webm_dash
17      3gp  176x144     12  1 │   55.79KiB   45k https │ mp4v.20.3       45k mp4a.40.2    0k 22k 144p, AND
269     mp4  256x144     15    │ ~214.75KiB  172k m3u8  │ avc1.4D400C    172k video only
160     mp4  256x144     15    │  135.08KiB  113k https │ avc1.4d400c    113k video only          144p, AND, mp4_dash
603     mp4  256x144     30    │ ~127.69KiB  102k m3u8  │ vp09.00.11.08  102k video only
278     webm 256x144     30    │   52.22KiB   44k https │ vp9             44k video only          144p, AND, webm_dash
229     mp4  426x240     30    │ ~386.34KiB  309k m3u8  │ avc1.4D4015    309k video only
133     mp4  426x240     30    │  294.27KiB  246k https │ avc1.4d4015    246k video only          240p, AND, mp4_dash
604     mp4  426x240     30    │ ~ 99.87KiB   80k m3u8  │ vp09.00.20.08   80k video only
242     webm 426x240     30    │   33.27KiB   28k https │ vp9             28k video only          240p, AND, webm_dash
230     mp4  640x360     30    │ ~553.39KiB  443k m3u8  │ avc1.4D401E    443k video only
134     mp4  640x360     30    │  349.59KiB  292k https │ avc1.4d401e    292k video only          360p, AND, mp4_dash
18      mp4  640x360     30  2 │ ~525.60KiB  420k https │ avc1.42001E    420k mp4a.40.2    0k 44k 360p, AND
605     mp4  640x360     30    │ ~255.57KiB  204k m3u8  │ vp09.00.21.08  204k video only
243     webm 640x360     30    │   75.55KiB   63k https │ vp9             63k video only          360p, AND, webm_dash
231     mp4  854x480     30    │ ~  1.10MiB  898k m3u8  │ avc1.4D401F    898k video only
135     mp4  854x480     30    │  849.41KiB  710k https │ avc1.4d401f    710k video only          480p, AND, mp4_dash
606     mp4  854x480     30    │ ~375.46KiB  300k m3u8  │ vp09.00.30.08  300k video only
244     webm 854x480     30    │  165.49KiB  138k https │ vp9            138k video only          480p, AND, webm_dash
22      mp4  1280x720    30  2 │ ~  1.82MiB 1493k https │ avc1.64001F   1493k mp4a.40.2    0k 44k 720p, AND
232     mp4  1280x720    30    │ ~  1.99MiB 1631k m3u8  │ avc1.4D401F   1631k video only
136     mp4  1280x720    30    │    1.60MiB 1366k https │ avc1.4d401f   1366k video only          720p, AND, mp4_dash
609     mp4  1280x720    30    │ ~756.27KiB  605k m3u8  │ vp09.00.31.08  605k video only
247     webm 1280x720    30    │  504.68KiB  420k https │ vp9            420k video only          720p, AND, webm_dash
270     mp4  1920x1080   30    │ ~  2.52MiB 2061k m3u8  │ avc1.640028   2061k video only
137     mp4  1920x1080   30    │    2.11MiB 1803k https │ avc1.640028   1803k video only          1080p, AND, mp4_dash
614     mp4  1920x1080   30    │ ~  1.29MiB 1057k m3u8  │ vp09.00.40.08 1057k video only
248     webm 1920x1080   30    │  965.31KiB  804k https │ vp9            804k video only          1080p, AND, webm_dash
[info] BaW_jenozKc: Downloading 1 format(s): 248+251
```

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

**Comment by pukkandan** [comment]
> If you wish to enforce this to all the formats (MPD and HLS), a little rewrite would be required

That would be nice

**Comment by Lesmiscore** [comment]
```
[info] Available formats for BaW_jenozKc:
ID      EXT  RESOLUTION FPS CH │   FILESIZE   TBR PROTO │ VCODEC          VBR ACODEC      ABR ASR MORE INFO
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
233     mp4  audio only        │                  m3u8  │ audio only          unknown             Default, IOS
234     mp4  audio only        │                  m3u8  │ audio only          unknown             Default, IOS
249-drc webm audio only      2 │   62.24KiB   52k https │ audio only          opus        52k 48k low, DRC, TVH, webm_dash
250-drc webm audio only      2 │   80.49KiB   67k https │ audio only          opus        67k 48k low, DRC, TVH, webm_dash
139     m4a  audio only      2 │   58.59KiB   48k https │ audio only          mp4a.40.5   48k 22k low, AND, m4a_dash
249     webm audio only      2 │   58.17KiB   48k https │ audio only          opus        48k 48k low, AND, webm_dash
250     webm audio only      2 │   76.07KiB   63k https │ audio only          opus        63k 48k low, AND, webm_dash
140-drc m4a  audio only      2 │  157.22KiB  130k https │ audio only          mp4a.40.2  130k 44k medium, DRC, TVH, m4a_dash
251-drc webm audio only      2 │  138.80KiB  115k https │ audio only          opus       115k 48k medium, DRC, TVH, webm_dash
140     m4a  audio only      2 │  154.06KiB  128k https │ audio only          mp4a.40.2  128k 44k medium, AND, m4a_dash
251     webm audio only      2 │  138.96KiB  116k https │ audio only          opus       116k 48k medium, AND, webm_dash
17      3gp  176x144     12  1 │   55.79KiB   45k https │ mp4v.20.3       45k mp4a.40.2    0k 22k 144p, AND
269     mp4  256x144     15    │ ~214.75KiB  172k m3u8  │ avc1.4D400C    172k video only          IOS
160     mp4  256x144     15    │  135.08KiB  113k https │ avc1.4d400c    113k video only          144p, AND, mp4_dash
603     mp4  256x144     30    │ ~127.69KiB  102k m3u8  │ vp09.00.11.08  102k video only          IOS
278     webm 256x144     30    │   52.22KiB   44k https │ vp9             44k video only          144p, AND, webm_dash
229     mp4  426x240     30    │ ~386.34KiB  309k m3u8  │ avc1.4D4015    309k video only          IOS
133     mp4  426x240     30    │  294.27KiB  246k https │ avc1.4d4015    246k video only          240p, AND, mp4_dash
604     mp4  426x240     30    │ ~ 99.87KiB   80k m3u8  │ vp09.00.20.08   80k video only          IOS
242     webm 426x240     30    │   33.27KiB   28k https │ vp9             28k video only          240p, AND, webm_dash
230     mp4  640x360     30    │ ~553.39KiB  443k m3u8  │ avc1.4D401E    443k video only          IOS
134     mp4  640x360     30    │  349.59KiB  292k https │ avc1.4d401e    292k video only          360p, AND, mp4_dash
18      mp4  640x360     30  2 │ ~525.60KiB  420k https │ avc1.42001E    420k mp4a.40.2    0k 44k 360p, AND
605     mp4  640x360     30    │ ~255.57KiB  204k m3u8  │ vp09.00.21.08  204k video only          IOS
243     webm 640x360     30    │   75.55KiB   63k https │ vp9             63k video only          360p, AND, webm_dash
231     mp4  854x480     30    │ ~  1.10MiB  898k m3u8  │ avc1.4D401F    898k video only          IOS
135     mp4  854x480     30    │  849.41KiB  710k https │ avc1.4d401f    710k video only          480p, AND, mp4_dash
606     mp4  854x480     30    │ ~375.46KiB  300k m3u8  │ vp09.00.30.08  300k video only          IOS
244     webm 854x480     30    │  165.49KiB  138k https │ vp9            138k video only          480p, AND, webm_dash
22      mp4  1280x720    30  2 │ ~  1.82MiB 1493k https │ avc1.64001F   1493k mp4a.40.2    0k 44k 720p, AND
232     mp4  1280x720    30    │ ~  1.99MiB 1631k m3u8  │ avc1.4D401F   1631k video only          IOS
136     mp4  1280x720    30    │    1.60MiB 1366k https │ avc1.4d401f   1366k video only          720p, AND, mp4_dash
609     mp4  1280x720    30    │ ~756.27KiB  605k m3u8  │ vp09.00.31.08  605k video only          IOS
247     webm 1280x720    30    │  504.68KiB  420k https │ vp9            420k video only          720p, AND, webm_dash
270     mp4  1920x1080   30    │ ~  2.52MiB 2061k m3u8  │ avc1.640028   2061k video only          IOS
137     mp4  1920x1080   30    │    2.11MiB 1803k https │ avc1.640028   1803k video only          1080p, AND, mp4_dash
614     mp4  1920x1080   30    │ ~  1.29MiB 1057k m3u8  │ vp09.00.40.08 1057k video only          IOS
248     webm 1920x1080   30    │  965.31KiB  804k https │ vp9            804k video only          1080p, AND, webm_dash
```

File: yt_dlp/extractor/youtube.py [hunk 1]
```diff
@@ -66,6 +66,8 @@
     variadic,
 )
 
+
+STRAMING_DATA_CLIENT_NAME = '@@youtubeplzdontusethis@@client_name'
```
Comment (pukkandan) [inline yt_dlp/extractor/youtube.py:70]:
```suggestion
STREAMING_DATA_CLIENT_NAME = '__yt_dlp_client'
```
is similar to what we use in jsinterp

File: yt_dlp/extractor/youtube.py [hunk 2]
```diff
@@ -3736,6 +3740,8 @@ def _extract_formats_and_subtitles(self, streaming_data, video_id, player_url, l
             if is_damaged:
                 self.report_warning(
                     f'{video_id}: Some formats are possibly damaged. They will be deprioritized', only_once=True)
+
+            client_name = (traverse_obj(query, ('c', 0, {str})) or fmt.get(STRAMING_DATA_CLIENT_NAME) or '')[0:3].upper()
```
Comment (pukkandan) [inline yt_dlp/extractor/youtube.py:3744]: With 3 chars, we can't tell `web_embedded` etc apart. Maybe use thw whole string?

Comment (Lesmiscore) [reply yt_dlp/extractor/youtube.py:3744]: The reason behind 3 characters is that it's narrow enough not to cause break lines in my console and it's long enough to identify which device is problematic, at first glance
And if we need the whole string, it should be done by editing code as needed

**Comment by pukkandan** [comment]
1. There is no distinction b/w web, web_embeeded etc
2. It's sometimes giving `clientName` and other times the friendly name yt-dlp uses

What do you think about this?
```diff
diff --git a/yt_dlp/extractor/youtube.py b/yt_dlp/extractor/youtube.py
index cfad8b325..397e68137 100644
--- a/yt_dlp/extractor/youtube.py
+++ b/yt_dlp/extractor/youtube.py
@@ -250,6 +250,11 @@ def _split_innertube_client(client_name):
     return client_name, base, variant[0] if variant else None


+def short_client_name(client_name):
+    parts = _split_innertube_client(client_name)[0].replace('embedscreen', 'e_s').split('_')
+    return ''.join(x[0] for x in parts).upper()
+
+
 def build_innertube_clients():
     THIRD_PARTY = {
         'embedUrl': 'https://www.youtube.com/',  # Can be any valid URL
@@ -3620,8 +3625,9 @@ def append_client(*client_names):
                     self.report_warning(
                         f'Skipping player response from {client} client (got player response for video "{pr_video_id}" instead of "{video_id}")' + bug_reports_message())
                 else:
-                    # set the client name in streamingData so that we can show it on -F
-                    (traverse_obj(pr, ('streamingData', {dict})) or {})[STREAMING_DATA_CLIENT_NAME] = client
+                    # Save client name for introspection later
+                    for f in traverse_obj(pr, ('streamingData', ('formats', 'adaptiveFormats'), ..., {dict})):
+                        f[STREAMING_DATA_CLIENT_NAME] = short_client_name(client)
                     prs.append(pr)

             # creator clients can bypass AGE_VERIFICATION_REQUIRED if logged in
@@ -3741,7 +3747,7 @@ def _extract_formats_and_subtitles(self, streaming_data, video_id, player_url, l
                 self.report_warning(
                     f'{video_id}: Some formats are possibly damaged. They will be deprioritized', only_once=True)

-            client_name = (traverse_obj(query, ('c', 0, {str})) or fmt.get(STREAMING_DATA_CLIENT_NAME) or '')[0:3].upper()
+            client_name = fmt.get(STREAMING_DATA_CLIENT_NAME)
             dct = {
                 'asr': int_or_none(fmt.get('audioSampleRate')),
                 'filesize': int_or_none(fmt.get('contentLength')),
@@ -3754,7 +3760,6 @@ def _extract_formats_and_subtitles(self, streaming_data, video_id, player_url, l
                     try_get(fmt, lambda x: x['projectionType'].replace('RECTANGULAR', '').lower()),
                     try_get(fmt, lambda x: x['spatialAudioType'].replace('SPATIAL_AUDIO_TYPE_', '').lower()),
                     throttled and 'THROTTLED', is_damaged and 'DAMAGED',
-                    # for convenience of seeing client name in -F without debugging
                     client_name,
                     delim=', '),
                 # Format 22 is likely to be damaged. See https://github.com/yt-dlp/yt-dlp/issues/3372
@@ -3833,7 +3838,7 @@ def process_manifest_format(f, proto, client_name, itag):

         subtitles = {}
         for sd in streaming_data:
-            client_name = (sd.get(STREAMING_DATA_CLIENT_NAME) or '')[0:3].upper()
+            client_name = sd.get(STREAMING_DATA_CLIENT_NAME)

             hls_manifest_url = 'hls' not in skip_manifests and sd.get('hlsManifestUrl')
             if hls_manifest_url:
```
It's compact and gives more info. Issue is that it's harder to read for those unfamilar with code
```
❯ yt-dlp --client=android,web,tv_embedded -qF BaW_jenozKc && yt-dlp --client=web_embedded,android_embedscreen -qF BaW_jenozKc && yt-dlp --client=mweb -qF BaW_jenozKc
ID      EXT  RESOLUTION FPS CH │   FILESIZE   TBR PROTO │ VCODEC        VBR ACODEC      ABR ASR MORE INFO
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
249-drc webm audio only      2 │   62.24KiB   52k https │ audio only        opus        52k 48k low, DRC, TE, webm_dash
250-drc webm audio only      2 │   80.49KiB   67k https │ audio only        opus        67k 48k low, DRC, TE, webm_dash
139     m4a  audio only      2 │   58.59KiB   48k https │ audio only        mp4a.40.5   48k 22k low, A, m4a_dash
249     webm audio only      2 │   58.17KiB   48k https │ audio only        opus        48k 48k low, A, webm_dash
250     webm audio only      2 │   76.07KiB   63k https │ audio only        opus        63k 48k low, A, webm_dash
140-drc m4a  audio only      2 │  157.22KiB  130k https │ audio only        mp4a.40.2  130k 44k medium, DRC, TE, m4a_dash
251-drc webm audio only      2 │  138.80KiB  115k https │ audio only        opus       115k 48k medium, DRC, TE, webm_dash
140     m4a  audio only      2 │  154.06KiB  128k https │ audio only        mp4a.40.2  128k 44k medium, A, m4a_dash
251     webm audio only      2 │  138.96KiB  116k https │ audio only        opus       116k 48k medium, A, webm_dash
17      3gp  176x144     12  1 │   55.79KiB   45k https │ mp4v.20.3     45k mp4a.40.2    0k 22k 144p, A
160     mp4  256x144     15    │  135.08KiB  113k https │ avc1.4d400c  113k video only          144p, A, mp4_dash
278     webm 256x144     30    │   52.22KiB   44k https │ vp9           44k video only          144p, A, webm_dash
133     mp4  426x240     30    │  294.27KiB  246k https │ avc1.4d4015  246k video only          240p, A, mp4_dash
242     webm 426x240     30    │   33.27KiB   28k https │ vp9           28k video only          240p, A, webm_dash
134     mp4  640x360     30    │  349.59KiB  292k https │ avc1.4d401e  292k video only          360p, A, mp4_dash
18      mp4  640x360     30  2 │ ~525.60KiB  420k https │ avc1.42001E  420k mp4a.40.2    0k 44k 360p, A
243     webm 640x360     30    │   75.55KiB   63k https │ vp9           63k video only          360p, A, webm_dash
135     mp4  854x480     30    │  849.41KiB  710k https │ avc1.4d401f  710k video only          480p, A, mp4_dash
244     webm 854x480     30    │  165.49KiB  138k https │ vp9          138k video only          480p, A, webm_dash
22      mp4  1280x720    30  2 │ ~  1.82MiB 1493k https │ avc1.64001F 1493k mp4a.40.2    0k 44k 720p, A
136     mp4  1280x720    30    │    1.60MiB 1366k https │ avc1.4d401f 1366k video only          720p, A, mp4_dash
247     webm 1280x720    30    │  504.68KiB  420k https │ vp9          420k video only          720p, A, webm_dash
137     mp4  1920x1080   30    │    2.11MiB 1803k https │ avc1.640028 1803k video only          1080p, A, mp4_dash
248     webm 1920x1080   30    │  965.31KiB  804k https │ vp9          804k video only          1080p, A, webm_dash
ID  EXT  RESOLUTION FPS CH │   FILESIZE   TBR PROTO │ VCODEC        VBR ACODEC      ABR ASR MORE INFO
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────
139 m4a  audio only      2 │   58.59KiB   48k https │ audio only        mp4a.40.5   48k 22k low, AES, m4a_dash
249 webm audio only      2 │   58.17KiB   48k https │ audio only        opus        48k 48k low, WE, webm_dash
250 webm audio only      2 │   76.07KiB   63k https │ audio only        opus        63k 48k low, WE, webm_dash
140 m4a  audio only      2 │  154.06KiB  128k https │ audio only        mp4a.40.2  128k 44k medium, WE, m4a_dash
251 webm audio only      2 │  138.96KiB  116k https │ audio only        opus       116k 48k medium, WE, webm_dash
17  3gp  176x144     12  1 │   55.79KiB   45k https │ mp4v.20.3     45k mp4a.40.2    0k 22k 144p, AES
160 mp4  256x144     15    │  135.08KiB  113k https │ avc1.4d400c  113k video only          144p, WE, mp4_dash
278 webm 256x144     30    │   52.22KiB   44k https │ vp9           44k video only          144p, WE, webm_dash
133 mp4  426x240     30    │  294.27KiB  246k https │ avc1.4d4015  246k video only          240p, WE, mp4_dash
242 webm 426x240     30    │   33.27KiB   28k https │ vp9           28k video only          240p, WE, webm_dash
134 mp4  640x360     30    │  349.59KiB  292k https │ avc1.4d401e  292k video only          360p, WE, mp4_dash
18  mp4  640x360     30  2 │ ~525.60KiB  420k https │ avc1.42001E  420k mp4a.40.2    0k 44k 360p, WE
243 webm 640x360     30    │   75.55KiB   63k https │ vp9           63k video only          360p, WE, webm_dash
135 mp4  854x480     30    │  849.41KiB  710k https │ avc1.4d401f  710k video only          480p, WE, mp4_dash
244 webm 854x480     30    │  165.49KiB  138k https │ vp9          138k video only          480p, WE, webm_dash
22  mp4  1280x720    30  2 │ ~  1.82MiB 1493k https │ avc1.64001F 1493k mp4a.40.2    0k 44k 720p, WE
136 mp4  1280x720    30    │    1.60MiB 1366k https │ avc1.4d401f 1366k video only          720p, WE, mp4_dash
247 webm 1280x720    30    │  504.68KiB  420k https │ vp9          420k video only          720p, WE, webm_dash
137 mp4  1920x1080   30    │    2.11MiB 1803k https │ avc1.640028 1803k video only          1080p, WE, mp4_dash
248 webm 1920x1080   30    │  965.31KiB  804k https │ vp9          804k video only          1080p, WE, webm_dash
ID  EXT  RESOLUTION FPS CH │   FILESIZE   TBR PROTO │ VCODEC        VBR ACODEC      ABR ASR MORE INFO
────────────────────────────────────────────────────────────────────────────────────────────────────────────────
249 webm audio only      2 │   58.17KiB   48k https │ audio only        opus        48k 48k low, M, webm_dash
250 webm audio only      2 │   76.07KiB   63k https │ audio only        opus        63k 48k low, M, webm_dash
140 m4a  audio only      2 │  154.06KiB  128k https │ audio only        mp4a.40.2  128k 44k medium, M, m4a_dash
251 webm audio only      2 │  138.96KiB  116k https │ audio only        opus       116k 48k medium, M, webm_dash
17  3gp  176x144     12  1 │   55.79KiB   45k https │ mp4v.20.3     45k mp4a.40.2    0k 22k 144p, M
160 mp4  256x144     15    │  135.08KiB  113k https │ avc1.4d400c  113k video only          144p, M, mp4_dash
278 webm 256x144     30    │   52.22KiB   44k https │ vp9           44k video only          144p, M, webm_dash
133 mp4  426x240     30    │  294.27KiB  246k https │ avc1.4d4015  246k video only          240p, M, mp4_dash
242 webm 426x240     30    │   33.27KiB   28k https │ vp9           28k video only          240p, M, webm_dash
134 mp4  640x360     30    │  349.59KiB  292k https │ avc1.4d401e  292k video only          360p, M, mp4_dash
18  mp4  640x360     30  2 │ ~525.60KiB  420k https │ avc1.42001E  420k mp4a.40.2    0k 44k 360p, M
243 webm 640x360     30    │   75.55KiB   63k https │ vp9           63k video only          360p, M, webm_dash
135 mp4  854x480     30    │  849.41KiB  710k https │ avc1.4d401f  710k video only          480p, M, mp4_dash
244 webm 854x480     30    │  165.49KiB  138k https │ vp9          138k video only          480p, M, webm_dash
22  mp4  1280x720    30  2 │ ~  1.82MiB 1493k https │ avc1.64001F 1493k mp4a.40.2    0k 44k 720p, M
136 mp4  1280x720    30    │    1.60MiB 1366k https │ avc1.4d401f 1366k video only          720p, M, mp4_dash
247 webm 1280x720    30    │  504.68KiB  420k https │ vp9          420k video only          720p, M, webm_dash
137 mp4  1920x1080   30    │    2.11MiB 1803k https │ avc1.640028 1803k video only          1080p, M, mp4_dash
248 webm 1920x1080   30    │  965.31KiB  804k https │ vp9          804k video only          1080p, M, webm_dash
```

Also, maybe we could add it only with `-v`?
```diff
-                      client_name,
+                      client_name if self.get_param('verbose') else None,
```

cc @coletdjnz

**Comment by pukkandan** [comment]
You never replied, @Lesmiscore

**Comment by Lesmiscore** [comment]
> 1. There is no distinction b/w web, web_embeeded etc
That's fine to me, but you don't like it?

> 2. It's sometimes giving `clientName` and other times the friendly name yt-dlp uses
Use the YouTube's client name as always?

**Comment by coletdjnz** [comment]
You could use the client integer number instead if it needs to be compact, though it's a bit less clear.

**Comment by Lesmiscore** [comment]
How are we supposed to get that in `_extract_player_responses` though? I have no idea actually

**Comment by pukkandan** [comment]
The code in https://github.com/yt-dlp/yt-dlp/pull/6254#issuecomment-1434751755 can be tweaked to pass any data about the client into where formats are built. We just need to decide how we want it to be displayed.

> You could use the client integer number instead if it needs to be compact, though it's a bit less clear.

Using numeric code is worse than abbreviation (my code) imo

**Pull request merged**